src/main/java/cubicchunks/asm/mixin/selectable/common/MixinWorldServer_UpdateBlocks.java

```diff
@@ -0,0 +1,157 @@
+/*
+ *  This file is part of Cubic Chunks Mod, licensed under the MIT License (MIT).
+ *
+ *  Copyright (c) 2015 contributors
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package cubicchunks.asm.mixin.selectable.common;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import cubicchunks.server.CubeProviderServer;
+import cubicchunks.util.CCEntitySelectors;
+import cubicchunks.util.Coords;
+import cubicchunks.util.CubePos;
+import cubicchunks.world.ICubicWorldServer;
+import cubicchunks.world.column.IColumn;
+import cubicchunks.world.cube.Cube;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.effect.EntityLightningBolt;
+import net.minecraft.entity.passive.EntitySkeletonHorse;
+import net.minecraft.init.Blocks;
+import net.minecraft.profiler.Profiler;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.WorldProvider;
+import net.minecraft.world.WorldServer;
+import net.minecraft.world.WorldType;
+import net.minecraft.world.chunk.Chunk;
+
+@Mixin(value = WorldServer.class, priority = 1001)
+public abstract class MixinWorldServer_UpdateBlocks implements ICubicWorldServer {
+
+    @Shadow
+    protected abstract void playerCheckLight();
+
+    @Inject(method = "updateBlocks", at = @At("HEAD"), cancellable = true)
+    public void onUpdateBlocks(CallbackInfo ci) {
```

Comment:
Could you instead attempt to redirect the right methods to no-ops here or using ModifyVariable? This overwrite would need updating whenever this method is changed and it's a big method. And it's likely that mods will attempt to modify it.